### DISCUSSION TITLE
how can i get browser url

### DISCUSSION THREAD

**Discussion by mcfly001** [coder/code-server#2632] [Q&A]
How to access browser URL in vscode plugin

**Comment by oxy** [comment]
What exactly do you mean by this? Can you be a little more specific?

**Reply by mcfly001** [reply]
Multiple people visit the page, and I need to determine the user information from the browser URL

**Comment by code-asher** [comment]
There isn't a way yet but we have some ideas here: #1936 #1510

**Reply by mcfly001** [reply]
have a demo? how to set it?

**Reply by code-asher** [reply]
There isn't a way to do it; neither of those issues have been implemented yet.
